packages/invoices/src/InboundShipment/DetailView/modals/InboundLineEdit/InboundLineEdit.tsx

```diff
@@ -208,9 +232,21 @@ export const InboundLineEdit: FC<InboundLineEditProps> = ({
                     centered
                     onChange={(_, v) => setCurrentTab(v)}
                   >
-                    <Tab value={Tabs.Batch} label={t('label.quantities')} />
-                    <Tab value={Tabs.Pricing} label={t('label.pricing')} />
-                    <Tab value={Tabs.Location} label={t('label.location')} />
+                    <Tab
+                      value={Tabs.Batch}
+                      label={`${t('label.quantities')} (⇧+1)`}
+                      tabIndex={-1}
+                    />
+                    <Tab
+                      value={Tabs.Pricing}
+                      label={`${t('label.pricing')} (⇧+2)`}
+                      tabIndex={-1}
+                    />
+                    <Tab
+                      value={Tabs.Location}
+                      label={`${t('label.location')} (⇧+3)`}
+                      tabIndex={-1}
+                    />
```

Comment:
The main motivation for doing this now, was that the tab order is a little strange when there are multiple rows as the first two columns are a different table.

If we add a 'focus' to the first cell of the 'tabs table' when we change tab, it might be quite nice? Not really sure.. but the alternative is to bring the columns. back into the table which means we have to manually make sure that the widths and stuff are aligned and it's a bit tedious 😁

Reply:
oh. heh. I commented about exactly that. yeah - a focus would be nice, I think.